README.md

```diff
@@ -15,6 +24,41 @@ The steps below describe what needs to be done to use Platoon for
 data-parallelism. The LSTM example in the folder 'example' was implemented
 following these steps and should be referred to for guidance.
 
+Timing of the LSTM example on 2 k80
+-----------------------------------
+
+The timing is about efficiency of computation, not efficiency of
+training.  So the parameter alpha is constant. The number of minibatch
+is fixed as the hyper-parameter. The sync is also fixed to be after 10
+mini-batch of computation.
+
+With 1 worker, it won't train well. This isn't recommended. This is
+there just to show the overhead of the EASGD implementation.  Normal
+is without this framework, also there for overhead evaluation.
+
+Normal | 1 GPU | 2 GPU | 3 GPU | 4 GPU
+-------+-------+-------+-------+-------
+ 870s  |  912s |  477s |  329s |  254s
+ 1.00x | 0.95x | 1.82x | 2.65x | 3.42x
+
+
+Real usage consideration
+------------------------
+
+The optimal (as in more efficient for learning) hyper-parameters are
+dependant on the number of workers.  At least, consider changing, the
+learning rate and the alpha parameter of EASGD.
+
+How to change the alpha hyper-parameter isn't clear. An alpha of 0.5
+for the LSTM example with 2 workers seem to have good training
+efficency for this model/dataset/hyper-parameter combination.
+
+Is 1/N a good guideline for alpha? 1 datapoint! 0.5 seems to work well
+for 2 workers, but not so well for 3 or 4 workers.
```

Comment:
Also(x2), the optimal alpha value might be influenced by the fact that when a worker does valid, it doesn't get the master parameters before or after. This might end up penalizing certain alpha values because alpha will control how much, when a worker is done validating (and it has 'outdated' parameters), it will drag the parameters back to their 'outdated' values.